doc/rst/language/spec/procedures.rst

```diff
@@ -576,70 +576,93 @@ Abstract Intents
+    |                         | ``const`` intent     |                         |                                                      |
+    | Type                    | meaning              | Default intent meaning  | Notes                                                |
+    +=========================+======================+=========================+======================================================+
+    | scalar types            |  ``const in``        | ``const in``            |                                                      |
+    |                         |                      |                         |                                                      |
+    | (``bool``,              |                      |                         |                                                      |
+    | ``int``, ``uint``,      |                      |                         |                                                      |
+    | ``real``, ``imag``,     |                      |                         |                                                      |
+    | ``complex``)            |                      |                         |                                                      |
+    +-------------------------+----------------------+-------------------------+------------------------------------------------------+
+    | string-like types       | ``const ref``        | ``const ref``           |                                                      |
+    |                         |                      |                         |                                                      |
+    | (``string``, ``bytes``) |                      |                         |                                                      |
+    +-------------------------+----------------------+-------------------------+------------------------------------------------------+
+    | ranges                  | ``const in``         | ``const in``            |                                                      |
+    +-------------------------+----------------------+-------------------------+------------------------------------------------------+
+    | domains / domain maps   | ``const ref``        | ``const ref``           |                                                      |
+    +-------------------------+----------------------+-------------------------+------------------------------------------------------+
+    | arrays                  | ``const ref``        | ``ref`` / ``const ref`` | see :ref:`Default_Intent_for_Arrays_and_Record_this` |
+    +-------------------------+----------------------+-------------------------+------------------------------------------------------+
+    | classes                 | ``const ref``        | ``const ref``           | see :ref:`Default_Intent_for_Arrays_and_Record_this` |
```

Comment:
I think you meant "records" here rather than "classes"

Reply:
D'oh!  Sloppy, brad, sloppy...